console-extensions.json

```diff
@@ -17,7 +21,44 @@
         "group": "kubevirt.io",
         "kind": "VirtualMachine",
         "version": "v1"
+      },
+      "dataAttributes": {
+        "data-quickstart-id": "qs-nav-virtualmachines",
+        "data-test-id": "virtualmachines-nav-item"
+      }
+    }
+  },
+  {
+    "type": "console.navigation/resource-ns",
+    "properties": {
+      "id": "virtualmachines",
+      "section": "virtualization",
+      "name": "%plugin__kubevirt-plugin~VirtualMachineInstances%",
+      "model": {
+        "group": "kubevirt.io",
+        "kind": "VirtualMachineInstance",
+        "version": "v1"
+      },
+      "dataAttributes": {
+        "data-quickstart-id": "qs-nav-virtualmachineinstances",
+        "data-test-id": "virtualmachineinstances-nav-item"
+      }
+    }
+  },{
+    "type": "console.navigation/href",
+    "properties": {
+      "id": "templates",
+      "section": "virtualization",
+      "name": "%plugin__kubevirt-plugin~Templates%",
+      "href": "/templates",
+      "namespaced": true,
```

Comment:
how will this handle this kind of bug: https://bugzilla.redhat.com/show_bug.cgi?id=2047186
with current architecture? something we should consider.

Reply:
This is the fix for that bug.
no more special namespaces for templates, you only see the namespace you are in